### ISSUE TITLE
[Suggestion] Sliders, Hinges, Rotators in neutral/free

### ISSUE THREAD

**Issue by Demozo** [tsunamayo/Starship-EVO#2131]
Having sliders etc. in a neutral mode where they each are affected by physics.
Where a slider on a 45 degree angle would having it's attached entity slide in the direction of gravitation, for example.

Or rotators could be use to make pendulums.

**Comment by tsunamayo** [comment]
Yes I want to add deeper physics for children entity. I will need it to add springs and more. It is on the roadmap! Cheers
